src/test/features/claim/routes/defendant-add.ts

```diff
@@ -0,0 +1,89 @@
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pageText = 'Do you want to add another defendant?'
+
+describe('Claim issue: is defendant addition page', () => {
+  beforeEach(() => {
+    mock.cleanAll()
+    draftStoreServiceMock.resolveRetrieve('legalClaim')
+  })
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', ClaimPaths.defendantAdditionPage.uri)
+
+    it('should render page when everything is fine', async () => {
+      idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'claimant')
+
+      await request(app)
+        .get(ClaimPaths.defendantAdditionPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .expect(res => expect(res).to.be.successful.withText(pageText))
+    })
+  })
+
+  describe('on POST', () => {
+    checkAuthorizationGuards(app, 'post', ClaimPaths.defendantAdditionPage.uri)
+
+    it('should render page when form is invalid and everything is fine', async () => {
+      idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'claimant')
+
+      await request(app)
+        .post(ClaimPaths.defendantAdditionPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .expect(res => expect(res).to.be.successful.withText(pageText, 'div class="error-summary"'))
+    })
+
+    it('should return 500 and render error page when form is valid and cannot save draft', async () => {
+      idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'claimant')
+      draftStoreServiceMock.rejectSave('legalClaim', 'HTTP error')
+
+      await request(app)
+        .post(ClaimPaths.defendantAdditionPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .send({
+          isAddDefendant: 'YES'
+        })
+        .expect(res => expect(res).to.be.serverError.withText('Error'))
+    })
+
+    it('should redirect to defendant rep address page when form is valid and user has selected yes', async () => {
+      idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'claimant')
+      draftStoreServiceMock.resolveSave('legalClaim')
+
+      await request(app)
+        .post(ClaimPaths.defendantAdditionPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .send({
+          isAddDefendant: 'YES'
+        })
+        .expect(res => expect(res).to.be.redirect.toLocation(ClaimPaths.defendantTypePage.uri))
+    })
+
+    it('should redirect to defendant add page when form is valid and user has selected no', async () => {
+      idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'claimant')
+      draftStoreServiceMock.resolveSave('legalClaim')
+
+      await request(app)
+        .post(ClaimPaths.defendantAdditionPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .send({
+          isAddDefendant: 'NO'
+        })
+        .expect(res => expect(res).to.be.redirect.toLocation(ClaimPaths.personalInjuryPage.uri))
```

Comment:
description of test says `should redirect to defendant add page` but its redirecting to personal injury.